pl_examples/full_examples/semantic_segmentation/models/unet/model.py

```diff
@@ -0,0 +1,42 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+from parts import DoubleConv, Down, Up
+
+
+class UNet(nn.Module):
```

Comment:
I would really like the idea of creating a model ZOO inside lightning, @PyTorchLightning/core-contributors ?